event-app-api/server.js

```diff
@@ -203,10 +204,16 @@ app.post("/event_popup", async (req, res) => {
   const startTime = req.body.startTime;
   const endTime = req.body.endTime;
 
+  const startTimeObj = new Date(startTime);
+  const startTimeEpoch = startTimeObj.getTime() / 1000;
+
+  const endTimeObj = new Date(endTime);
+  const endTimeEpoch = endTimeObj.getTime() / 1000;
+
```

Comment:
do you not need to change the start/end time data type in the schema for your table? before it was string, now it's an int right?